src/test/java/FactoryBoyTest.java

```diff
@@ -2,33 +2,75 @@
 import me.carlosribeiro.factory_boy.FactoryBoy;
 import me.carlosribeiro.factory_boy.FactoryBuilder;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
 import static org.junit.Assert.*;
 
+import java.math.BigDecimal;
+import java.util.List;
+
 class SampleClass {
-    private String name;
+  private String name;
+  private Integer age;
+  private BigDecimal value;
+
+  public String getName() {
+    return this.name;
+  }
+
+  public void setName(String name) {
```

Comment:
Please remove this setter. This will prove that the lib can even set the attribute without the model have a setter defined.